src/Repositories/WSDLCacheRepository.php

```diff
@@ -0,0 +1,43 @@
+<?php namespace SIVI\LaravelAFD\Repositories;
+
+use Carbon\Carbon;
+use Illuminate\Support\Facades\Cache;
+use SIVI\AFDConnectors\Connectors\Contracts\TIMEConnector;
+
+class WSDLCacheRepository implements \SIVI\AFDConnectors\Repositories\Contracts\WSDLCacheRepository
+{
+    public function __construct()
+    {
+        $this->app->resolving(TIMEConnector::class, function ($timeConnector, $app) {
+            $timeConnector->setCacheRepository($app->make(\SIVI\AFDConnectors\Repositories\Contracts\WSDLCacheRepository::class));
+        });

```

Comment:
Dit hoort in de ServiceProvider.

Reply:
oh waarin? boot / register / provides of ?

Reply:
> As mentioned previously, within the register method, you should only bind things into the service container. 

Zoals bijvoorbeeld een `resolving` binding.

Reply:
Had ik toch precies verkeerd ingeschat, ik dacht boot..
> Otherwise, you may accidentally use a service that is provided by a service provider which has not loaded yet.